main.go

```diff
@@ -13,6 +13,7 @@ const pluginPrefix = "BUILDKITE_PLUGIN_BUILDPIPE_"
 type Config struct {
 	Projects []Project     `yaml:"projects"`
 	Steps    []interface{} `yaml:"steps"`
+	Env      map[string]string `yaml:"env"`
```

Comment:
Nit: we should make the third column line up

Reply:
done ;)